src/screens/ProfileScreen/index.js

```diff
@@ -595,6 +599,7 @@ const ProfileScreen = ({route}) => {
   };
 
   const onDeletePost = async () => {
+    setIsOptionModalOpen(false);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isPostOptionModalOpen` state is being set to false when a post is deleted. Ensure this state change doesn't cause any unexpected side effects.
```diff
+ setIsOptionModalOpen(false);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->